### ISSUE TITLE
nvda does not read text selection in excel dialog

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#8030]
### Steps to reproduce:
Open excel.
Press alt
Right arrow until formulas
press tab until you enter function and press enter.
Now the focus will be on an editable field.
Enter a text and select.
Press nvda + shift + s so that nvda reads the selected text.

### Actual behavior:
nvda informs you that there is no text selected.

expected
The nvda should read the selected text.

### System configuration:
NVDA version:
next 14869

NVDA Installed or portable:
instaled
Other information:
> Example: Running in a VM

Windows version:
windows 7

Name and version of other software in use when reproducing the issue:
office 2013

### Other questions:

Does the issue still occur after restarting your PC?
yes

Have you tried any other versions of NVDA?
> Please list them and the result
no

**Comment by Adriani90** [comment]
I can reproduce this in MS Excel 2010 and 365 with NVDA Alpha 16691,10950C87.
cc: @lukaszgo1

**Comment by Adriani90** [comment]
I wonder if #8414 would improve this? cc: @leonardder
